tensorflow_datasets/scripts/refactoring/refactoring_dataset.py

```diff
@@ -0,0 +1,178 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
```

Comment:
Note: You can remove this as it is automatically added when merging.

Reply:
Ok, will remove

Reply:
Done